thirdeye/thirdeye-frontend/app/utils/yaml-tools.js

```diff
@@ -29,6 +28,20 @@ rules:
     type: DATA_SLA              # Alert if data is missing.
     params:
       sla: 3_DAYS               # Data is missing for 3 days since last availability
+
+# You can mention a cron to allow this detection to be run periodically
+# For example, the cron below would execute every 5 minutes.
+# cron: "0 0/5 * 1/1 * ? *"
+
+# Backfill detections
+# You can use the backfillStart parameter to denote how far back you want the anomalies
+# to be detected.
+# A valid entry is a timestamp value in millis.
+# 
+# - By default, (if not mentioned), the lookback is 30 days
+# - A value of 0 (zero) implies complete backfill till the start of data. As shown below.
+# backfillStart: 0 
+
```

Comment:
Hey @akshayrai 

I understand your point. The problem is that it is very hard for a regular user to get started on ThirdEye. These configurations are buried in the doc somewhere on a FAQ or you have to go through the code. I didn't know that there is a capability to add cron until someone else told me.

I think a YAML interface may be good for advanced users who can have configurations saved in files/elsewhere. However for a regular user,  it might help to just have a simple UI with text boxes and dropdowns etc which have hints/prepopulated and are easy to use without the need for documentation.

I added to the template here so that the user doesn't need to go find information elsewhere but is able to see it right within the app. I agree this makes the template more complex and that's the reason why I pushed it down at the bottom. Let me know what you think and I can revert the template changes.

Reply:
I understand. We too discussed this idea in the past, but the issue with this approach is that - There are several basic alert and subscription parameters which we might want to add to the template and this will simply keep growing over time making the config bigger and harder to read.

To help with this problem, the alternative we have taken is,
a. A link is provided in the create alert UI  (at the top) that should point to the documentation on the alert configs. (I can work on OS'ing this wiki if it isn't already)
b. Eventually, we should have a full form UI through which all the basic features should be surfaced. The yaml config is more for advanced users who can refer to a doc and update the configs as required.

So, it would be better to keep the template simple and put the additional settings in the doc.